### PR TITLE
Query params

### PR DESCRIPTION
**Pull request by richardgoater**
Hope you like this one dude, we could possibly add some validation for the params

### CODE REVIEW

**Comment by theosanderson** [comment]
Sorry for the delay! This is exceptionally cool and looks great to me!

**Comment by theosanderson** [comment]
Things we could think about in the future:
- persisting map position: "I want to link to a zoomed-in version of London"
- persisting "stacked/unstacked" status

**Comment by theosanderson** [comment]
Super cool. In case it wasn't clear these comments weren't demands for changes (but the changes are amazing!)

**Comment by richardgoater** [comment]
Thanks and no worries, I thought they were good suggestions and might as well get them done now :+1:

File: src/components/Chloropleth.jsx [hunk 1]
```diff
@@ -77,28 +91,37 @@ const ColourBar = ({ dmin, dmax, scale, type, className, percentage }) => {
...
-    } else if (newViewport.longitude > bounds.maxLongitude) {
-      newViewport.longitude = bounds.maxLongitude
-    }
-    if (newViewport.latitude < bounds.minLatitude) {
-      newViewport.latitude = bounds.minLatitude
-    } else if (newViewport.latitude > bounds.maxLatitude) {
-      newViewport.latitude = bounds.maxLatitude
+  useEffect(() => {
+    if (lat !== viewport.latitude || lon !== viewport.longitude || zoom !== viewport.zoom) {
+      console.log(viewport, lat, lon, zoom)
```
Comment (theosanderson) [inline src/components/Chloropleth.jsx:105]: FYI just noticed this lurking, not that it matters so much

**Comment by richardgoater** [comment]
hopefully this change didn't break anything, then I think we can merge 👍🏽

**Comment by theosanderson** [comment]
This is incredibly cool!! LGTM (with the caveat that some bits are a bit beyond me :) )

**Review by theosanderson — APPROVED** [review]

**Comment by richardgoater** [comment]
I'd be happy to talk through it, that would probably highlight some issues and make it better!

**Pull request merged**